tests/functions/test_origin_request.py

```diff
@@ -609,83 +608,46 @@ def test_origin_request_cookie_uri(
     ]
     assert request["headers"]["set-cookie"] == [
         {
-            "value": "CloudFront-Key-Pair-Id=K1MOU91G3N7WPY; Secure; "
-            "HttpOnly; SameSite=lax; Domain=ex.cloudfront.net; "
-            "Path=/content/; Max-Age=43200"
+            "value": "CloudFront-Key-Pair-Id=XXXXXXXXXXXXXX; Secure; "
+            "HttpOnly; SameSite=lax; Domain=localhost:8049; "
+            "Path=/content/; Max-Age=1800"
         },
         {
             "value": "CloudFront-Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaH"
-            "R0cHM6Ly9leC5jbG91ZGZyb250Lm5ldC9jb250ZW50LyoiLCJDb25kaXRpb24iOns"
-            "iRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjF9fX1dfQ__; Secure; "
-            "HttpOnly; SameSite=lax; Domain=ex.cloudfront.net; "
-            "Path=/content/; Max-Age=43200"
+            "R0cDovL2xvY2FsaG9zdDo4MDQ5L2NvbnRlbnQvKiIsIkNvbmRpdGlvbiI6eyJEYXR"
+            "lTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTY0NDk3MTQwMH19fV19; "
+            "Secure; HttpOnly; SameSite=lax; Domain=localhost:8049; "
+            "Path=/content/; Max-Age=1800"
```

Comment:
I think this significant reduction of Max-Age is pointing at another issue.  It is not desired that this change should make the cookies expire faster than they used to.

This *could* just be a test data issue, but I think it's a real issue which again has to be fixed on the exodus-gw side.  See: https://github.com/release-engineering/exodus-gw/pull/612#pullrequestreview-1693573305